core/server/translations/en.json

```diff
@@ -371,11 +371,7 @@
             },
             "icons": {
                 "missingFile": "Please select an icon.",
-                "fileSizeTooLarge": "Please select an icon file smaller than 100kb.",
-                "iconNotSquare": "The icon needs to be a square.",
-                "fileTooLarge": "Please select an icon file smaller than 1000px.",
-                "fileTooSmall": "Please select an icon file larger than 32px.",
-                "invalidFile": "Please select a valid icon file.",
+                "invalidFile": "Blog icon must be a square .ico or .png file between 32px – 1,000px, under 100kb.",
```

Comment:
This is from @JohnONolan here: https://github.com/TryGhost/Ghost/issues/7688#issuecomment-280575300